### DISCUSSION TITLE
Tindie shipping unassembled boards despite ordering a pre-assembled one.

### DISCUSSION THREAD

**Discussion by iLoveAndyBaker** [phil-barrett/PicoCNC#6] [General]
Ordered the pre-assembled, soldered header board, Tindie shipped parts in a bag but I paid full assembled price.  FYI. Has this happened to anybody else?

**Comment by phil-barrett** [comment]
I am sorry you are disappointed but we try to be very clear on what you get.  The Tindie product page only has two options: 

- Assembly Kit 
- No Assembly kit, source connectors yourself

The assembly kit is discussed in the description section:

> Assembly Kit.
> We are offer an optional assembly kit. It contains all the connectors needed to make your PicoCNC ready to install in your CNC machine. You will need to solder these in place.

If you wish, please return the package and we will refund your purchase price.

**Comment by iLoveAndyBaker** [comment]
I misunderstood - I assumed assembly kit meant assembled kit. My mistake. Thank you for taking the time to clear this up for me.
